recipes/easygdf/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "easygdf" %}
+{% set version = "2.0.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/easygdf-{{ version }}.tar.gz
+  sha256: 8d4c82e68a2247be8bd78cd47314795cc12594ea4b36877820c33deca2418e58
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - setuptools
+    - python >=3.6
+  run:
+    - numpy
+    - setuptools
```

Comment:
Since `setuptools` is only used by `setup.py`, you don't need it listed under `run`.

Reply:
Ooops, yeah. I forget about that one.